### PR TITLE
fragment: convert to std::future

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

**Comment by eugene-babichenko** [comment]
waiting for https://github.com/input-output-hk/jormungandr/pull/1744 to be merged

**Comment by rinor** [comment]
Can we merge this one? https://github.com/input-output-hk/jormungandr/pull/1744 is in.

**Comment by eugene-babichenko** [comment]
Waiting for a review before merging it.

File: jormungandr/src/utils/task.rs [hunk 1]
```diff
@@ -269,7 +269,7 @@ impl TokioServiceInfo {
                 let t_now = Instant::now();
                 interval.tick().await;
                 let t_last = Instant::now();
-                let elapsed = t_now.duration_since(t_last);
+                let elapsed = t_last.duration_since(t_now);
```
Comment (vincenthz) [inline jormungandr/src/utils/task.rs]: oopsie :)

**Pull request merged**